<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setup</class>
 <widget class="QWidget" name="setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="interval_title_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Interval</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="back_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="volume_title_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>VOLUME</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="date_year_highlight_label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>190</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_day_highlight_label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_hour_highlight_label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QLabel" name="interval_symbol_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>41</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string> s</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>20</width>
     <height>161</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="date_month_label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>DEC</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="date_year_label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>190</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>2021</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="time_ampm_label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>370</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>PM</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="time_minute_label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>55</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="volume_data_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="volume_highlight_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_day_label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>01</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="interval_data_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>25</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="time_miute_highlight_label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="time_hour_label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>07</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="startup_title_label">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Setup</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="down_button_label">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>300</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="time_ampm_highlight_label">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>370</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_month_highlight_label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>20</width>
     <height>161</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="interval_highlight_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QLabel" name="up_button_label">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>140</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="image_screen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="date_day_title_label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Day</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="date_month_title_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Month</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="date_year_title_label">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="time_hour_title_label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>310</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Hour</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="time_minute_title_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Minute</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="time_ampm_title_label">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>310</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>AM/PM</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="serial_title_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>S/N:</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="serial_data_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>550</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>90A25EE11</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <zorder>image_screen</zorder>
  <zorder>interval_title_label</zorder>
  <zorder>back_label</zorder>
  <zorder>volume_title_label</zorder>
  <zorder>date_year_highlight_label</zorder>
  <zorder>date_day_highlight_label</zorder>
  <zorder>time_hour_highlight_label</zorder>
  <zorder>interval_symbol_label</zorder>
  <zorder>line_3</zorder>
  <zorder>date_month_label</zorder>
  <zorder>line_4</zorder>
  <zorder>date_year_label</zorder>
  <zorder>time_ampm_label</zorder>
  <zorder>time_minute_label</zorder>
  <zorder>volume_data_label</zorder>
  <zorder>volume_highlight_label</zorder>
  <zorder>date_day_label</zorder>
  <zorder>interval_data_label</zorder>
  <zorder>time_miute_highlight_label</zorder>
  <zorder>line</zorder>
  <zorder>time_hour_label</zorder>
  <zorder>startup_title_label</zorder>
  <zorder>down_button_label</zorder>
  <zorder>time_ampm_highlight_label</zorder>
  <zorder>date_month_highlight_label</zorder>
  <zorder>line_2</zorder>
  <zorder>interval_highlight_label</zorder>
  <zorder>up_button_label</zorder>
  <zorder>date_day_title_label</zorder>
  <zorder>date_month_title_label</zorder>
  <zorder>date_year_title_label</zorder>
  <zorder>time_hour_title_label</zorder>
  <zorder>time_minute_title_label</zorder>
  <zorder>time_ampm_title_label</zorder>
  <zorder>serial_title_label</zorder>
  <zorder>serial_data_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
